exporter/local/export.go

```diff
@@ -149,6 +152,41 @@ func (e *localExporterInstance) Export(ctx context.Context, inp *exporter.Source
 			}
 
 			fs := fsutil.NewFS(src, walkOpt)
+
+			attestations, err = attestation.Unbundle(ctx, session.NewGroup(sessionID), inp.Refs, attestations)
+			if err != nil {
+				return err
+			}
+			stmts, err := attestation.Extract(ctx, session.NewGroup(sessionID), inp.Refs, attestations, nil)
+			if err != nil {
+				return err
+			}
+			if len(stmts) > 0 {
+				stmtFS := staticfs.NewFS()
+
+				names := map[string]struct{}{}
+				for i, stmt := range stmts {
+					dt, err := json.Marshal(stmt)
+					if err != nil {
+						return errors.Wrap(err, "failed to marshal attestation")
+					}
+
+					name := path.Base(attestations[i].Path)
```

Comment:
I think there should be a way to set prefix for this name with attestation opts (usually binary name that is exported with `type=local`).